shaku_derive/src/parser/property_from_field.rs

```diff
@@ -47,10 +47,26 @@ impl Parser<Property> for Field {
                             if has_default {
                                 Ok(PropertyDefault::NotProvided)
                             } else {
-                                Err(Error::new(
-                                    attr.span(),
-                                    format!("Unknown attribute: 'shaku{}'", attr.tokens),
```

Comment:
I think you could replace the format call with `"Unknown attribute"` to keep this simple. The span should indicate to the user where the error is.